schema.sql

```diff
@@ -9,3 +9,27 @@ CREATE TABLE animals (
 	weight_kg  decimal,
 	species varchar(50)
 );
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INTEGER
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
+ADD COLUMN species_id INTEGER REFERENCES species (id);
+
+ALTER TABLE animals
+ADD COLUMN owner_id INTEGER REFERENCES owners (id);
+
+
+
+SELECT * FROM animals;
```

Comment:
- [Optional] Dear @AleWaweru, Please remember everything that is a query that helps to analyze data should be saved in the queries.sql file. Hence the `select` statement should be written in the `queries.sql` file.

Reply:
Done.